style.css

```diff
@@ -698,11 +709,15 @@ h4 {
   background-color: #017243;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 930px) {
+  .body {
+    width: 100%;
+  }
+
```

Comment:
- [ ] You have done well to ensure that you have the desktop version of the application 👏 . However, as the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements) suggest, kindly consider using media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px) ☺️ 
- [ ] Additionally, I have noticed the horizontal scroll in your desktop application. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞 .